### PR TITLE
Replace jasmine/PhantomJS with Jest

### PR DESCRIPTION
**Pull request by aliuk2012**
Even though under the hood Jest still uses jasmine its still the new standard framework for testing JS. Jest includes JSDOM (A JavaScript implementation of various web standards, for use with Node.js) which simulates the browser this is ideal for unit testing client-side javascript without spinning up a full-blown browser.

What this changes:
- removes jasmine/PhantomJS
- adds and configures js
- configs StandardJS to include the` spec` folder when it lints our js files.
- fixes any new linting errors identified.
- wip analytics specs, we will revisit these tests when we come to implementing the new cookie banner and re-enabling GA.
- fixes two  GitHub identified High severity security vulnerabilities (uglify-js and handlebars)

## Upglify-js
image
## Handlebars
image

ticket: https://trello.com/c/qihIdbHM

### CODE REVIEW

**Comment by lfdebrux** [comment]
Does this mean we can remove `gulp-jasmine-phantom` from this repo as well?

**Comment by aliuk2012** [comment]
Yes, Will be done in a separate PR.

**Comment by lfdebrux** [comment]
Is this PR needed then? Would save a separate release cycle :D

**Comment by aliuk2012** [comment]
🤔  alrighty then 👍

**Comment by aliuk2012** [comment]
Based on https://github.com/alphagov/digitalmarketplace-buyer-frontend/pull/973

File: spec/javascripts/pii.spec.js [hunk 1]
```diff
@@ -1,139 +1,160 @@
...
 
-      obj = pii.stripPII(obj)
-      expect(obj).toEqual(strippedObj)
+      const results = pii.stripPII(obj)
+      expect(results).toEqual(strippedObj)
     })
 
-    it("strips all PII from arrays", function() {
-      var arr = [
+    it('trips all PII from arrays', () => {
```
Comment (katstevens) [inline spec/javascripts/pii.spec.js]: Typo - `trips`

Comment (aliuk2012) [reply spec/javascripts/pii.spec.js]: good catch 😄 👍

**Pull request merged**